Description: Honor LDFLAGS from the environment
 In Debian, hardened build needs LDFLAGS to be taken into account.
Author: Frederic Bonnard <frediz@linux.vnet.ibm.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/external/gard/rules.mk
+++ b/external/gard/rules.mk
@@ -28,7 +28,7 @@
 	$(CC) $(CFLAGS) -c $< -o $@
 
 $(EXE): $(OBJS)
-	$(CC) $(CFLAGS) $^ -o $@
+	$(CC) $(LDFLAGS) $(CFLAGS) $^ -o $@
 
 install: all
 	install -D gard $(DESTDIR)$(sbindir)/opal-gard
